ui_curses.c

```diff
@@ -2387,6 +2435,10 @@ static void init_all(void)
 
 	init_curses();
 
+	// enable bracketed paste (will be ignored if not supported)
```

Comment:
Fine by me. There's also `tgetstr("BE", ...)` but it's very new and not in my rxvt-unicode terminfo.

Reply:
Yep, I found the same on Fedora 38's nurses. Plus, this is how nano does it.